Buildings/Experimental/DHC/CentralPlants/Cooling/Controls/ChillerStage.mo

```diff
@@ -198,39 +198,16 @@ First implementation.
 </ul>
 </html>",
       info="<html>
-<p>
-This model implements the staging control logic (shown in the state graph below) of 
-two chillers according to the measured total cooling load. 
-The control logic is as follows:
-</p>
+<p>This model implements the staging control logic (shown in the state graph below) of two chillers according to the measured total cooling load. The control logic is as follows: </p>
```

Comment:
Such long lines need to be avoided.

Reply:
I meant all such long lines in the documentation need to be avoided (not only this one). You could simply hit enter where you wanted to start a new line.

Reply:
Should I go through all the model documentations and hit enter for long lines?